python/bin/amici_import_petab.py

```diff
@@ -199,18 +202,44 @@ def constant_species_to_parameters(sbml_model):
 
     # Remove from reactants and products
     for reaction in sbml_model.getListOfReactions():
-        for speciesId in transformables:
+        for species_id in transformables:
             # loop, since removeX only removes one instance
-            while reaction.removeReactant(speciesId):
+            while reaction.removeReactant(species_id):
                 pass
-            while reaction.removeProduct(speciesId):
+            while reaction.removeProduct(species_id):
                 pass
-            while reaction.removeModifier(speciesId):
+            while reaction.removeModifier(species_id):
                 pass
 
     return transformables
 
 
+def constant_species_to_parameters(sbml_model: 'libsbml.Model'):
+    """Convert constant species in the SBML model to constant parameters.
+
+    This can be used e.g. for setting up models with condition-specific
+    constant species for PEtab, since there it is not possible to specify
+    constant species in the condition table.
+
+    Arguments:
+        sbml_model: libsbml model instance
+
+    Returns:
+        List of IDs of SBML species that have been turned into constants
+
+    Raises:
+
+    """
+    transformables = []
```

Comment:
ok I guess you just wrote a function for converting any species to parameters, and extracted the checking of which species are constant to here. seems fine.